lib/addons/neuron/neuron-monitor.yaml

```diff
@@ -0,0 +1,85 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: neuron-monitor
+  namespace: kube-system
```

Comment:
I assume namespace should be a configurable parameter.

Reply:
Adressed